packages/atomic/src/components/facets-v1/atomic-facet/atomic-facet.tsx

```diff
@@ -31,6 +31,7 @@ import {
 } from '../facet-search/facet-search-utils';
 import {BaseFacet} from '../facet-common';
 import {FacetValueLabelHighlight} from '../facet-value-label-highlight/facet-value-label-highlight';
+import {Schema, StringValue} from '@coveo/bueno';
```

Comment:
Bueno is getting used in other projects by the way :) 

I've used it heavily for the CLI push commands, when we need to parse and validate arbitrary JSON documents before pushing it to Coveo cloud.

Works great 👌